Content.Client/CharacterInterface/CharacterInterfaceSystem.cs

```diff
@@ -49,6 +49,7 @@ private void OnComponentInit(EntityUid uid, CharacterInterfaceComponent comp, Co
                 return;
 
             comp.Window = new CharacterInterfaceComponent.CharacterWindow(comp.UIComponents);
+            comp.Window.MinSize = (545, 400);
```

Comment:
```suggestion
            comp.Window = new CharacterInterfaceComponent.CharacterWindow(comp.UIComponents)
            {
                MinSize = (545, 400);
            };.
```
If this works then this is how we usually do it.